components/inputNumber.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode, ReactNodeArray } from "react";
+import NumberFormat, { NumberFormatProps } from "react-number-format";
+
+export default function InputNumber({
+  children = "$BEPRO",
+  placeholder = "0",
+  ...params
+}: NumberFormatProps) {
+  return (
+    <div className="input-group">
+      <NumberFormat {...params} placeholder={placeholder} />
+      <span className="input-group-text text-white-50 p-small">{children}</span>
```

Comment:
Apparently, this span render a thing called 'tick' or 'symbol' (related to Currency Symbol/**$BEPRO** in this case). Would you not find interesting to define it through `[symbol]` instead of `[children]`?

Reply:
Done. -> 741c2a7. does this solve?